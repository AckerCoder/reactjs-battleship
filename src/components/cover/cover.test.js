import React from "react";
import { shallow } from "enzyme";
import Cover from "./cover.component";

it("Expect to Match Snapshot", () => {
  expect(shallow(<Cover />)).toMatchSnapshot();
});

it("Expect to render CustomInput Component", () => {
  const mockChildren = "Children";
  expect(shallow(<Cover>{mockChildren}</Cover>).length).toEqual(1);
});
