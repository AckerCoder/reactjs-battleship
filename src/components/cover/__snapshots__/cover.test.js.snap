// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to Match Snapshot 1`] = `
<div
  className="cover"
>
  <img
    alt="logo"
    className="logo"
    src="logo.svg"
  />
  <h1
    className="title"
  />
</div>
`;
